developer-guide/create-new-module.md

```diff
@@ -0,0 +1,284 @@
+# Creating a new module
+
+You can create a new module in IceHrm without putting much effort. Let's create a sample module called 'rooms'.
+
+There are 2 types of modules in IceHrm. Here we are creating an admin module. There are user modules as well. You need to choose `admin` or `module` folders inside `core` and `web` according to the module type you are creating.
+
+## Create Migration
+
+First we need to create a migration to add a new database table called 'MeetingRooms'.
+
+Create a new php file inside `core -> migrations` and name it accordingly.
+
+Write sql query to create the new table.
+
+```
+<?php
+   namespace Classes\Migration;
+   
+   class v20200331_270009_add_meeting_rooms_table extends AbstractMigration {
+   
+       public function up(){
+   
+           $sql = <<<'SQL'
+   create table `MeetingRooms` (
+   `id` bigint(20) NOT NULL AUTO_INCREMENT,
+   `name` varchar(100),
+   `description` varchar(500),
+   `meeting_link` varchar(500),
+   `departments` varchar(250),
+   `teams` varchar(250),
+   primary key (`id`),
+   `updated` datetime default NULL,
+   `created` datetime default NULL
+   ) engine=innodb default charset=utf8;
+   SQL;
+   
+         return $this->executeQuery($sql);
+   
+       }
+       public function down(){
+           return true;
+       }
+   }
+
+```
+
+Add the file name of this migration to the top of `list.php` file inside the same directory.
+
+Migration will run automatically and create the table.
+
+### Create model class
+
+Create a folder called 'Rooms' inside `core->src` and create the model class inside. We'll name it as 'MeetingRoom.php'. We can provide module access here.
+
+Its better to maintain the existing folder structure of the project throughout creating this new module. So refer the existing modules.
+
+
+```
+<?php
+   namespace Rooms\Common\Model;
+   
+   use Classes\ModuleAccess;
+   use Model\BaseModel;
+   
+   class MeetingRoom extends BaseModel
+   {
+       public $table = 'MeetingRooms';
+   
+       public function getAdminAccess()
```

Comment:
Explain this method, we need to allow Admins to 
get = see a list of meeting rooms
element = see a single meeting room
save = save and update a meeting room 
delete = delete a meeting room